build.sbt

```diff
@@ -10,7 +8,13 @@ version                    := "1.0.2"
 
 scalaVersion               := "2.11.1"
 
-utest.jsrunner.Plugin.utestJsSettings
+lazy val root = (project in file(".")).enablePlugins(ScalaJSPlugin)
```

Comment:
You forgot to put this line as the first line after the imports. Basically where `scalaJSSettings` stood previously.